src/frontend/src/lib/types/qr-code.ts

```diff
@@ -0,0 +1,8 @@
+export type QrResult = 'success' | 'cancelled' | 'token_incompatible';
+
+export type QrResponse = {
+	result: QrResult;
+	identifier?: string;
```

Comment:
`identifier` is probably a old reference to the old ICP account idenfitier, should we already use a more generic term?

Reply:
I will use `destination` since it is the standard way of defining the construction of a generic transaction QR code and it matches the rest of the code.